bootcampers-app/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_BACKEND_URL="https://bootcampersproject.herokuapp.com"
```

Comment:
all looks great we just need to make sure there is only one / between .com and users :)

Reply:
I have checked it during the tests. Thanks for remind me